src/2048-expert/index.js

```diff
@@ -223,18 +229,38 @@ function checkWin() {
   }
 }
 
-// a loss happens when all cells are full
+// a loss happens when all cells are full and they are not mergeable 
 function checkLost() {
+  if (checkisFull()) {
+    // check if any tiles are mergeable
+    for (let i = 0; i < totalCell-6; i++) {
+      // check vertical
+      if((cells[i].innerHTML == cells[i+6].innerHTML)){
+        // the user did not lose the game
```

Comment:
don't we need to check that at least **two** cells are mergable for them to not lose the game?

Reply:
It depends on how we want to implement the game. If the game is still playable after no empty cells are left, I think the game can continue to be played after one block left as well.